README.md

```diff
@@ -29,9 +29,8 @@ See a complete working example in the [roots-example-project.com repo](https://g
 
 ## Installation
 
-1. Clone the git repo - `git clone https://github.com/roots/bedrock.git`
-2. Run `composer install`
-3. Copy `.env.example` to `.env` and update environment variables:
+1. Create fresh project with this repo - `composer create-project roots/bedrock wordpress "1.6.*"`
```

Comment:
Sorry I meant this should just be `composer create-project roots/bedrock`. No need for `wordpress "1.6.*"`. Otherwise we'd always have to update the README with new versions.